libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -741,10 +756,8 @@ void PbftManager::proposeBlock_() {
   auto [round, period] = getPbftRoundAndPeriod();
   LOG(log_dg_) << "PBFT value proposal state in round: " << round << ", period: " << period;
 
-  if (round == 1 || next_votes_manager_->haveEnoughVotesForNullBlockHash()) {
-    if (round > 1) {
-      LOG(log_nf_) << "Previous round " << round - 1 << " had next voted NULL_BLOCK_HASH";
-    }
+  if (round == 1 || previous_round_next_voted_null_block_hash_) {
+    LOG(log_nf_) << "Previous round " << round - 1 << " had next voted NULL_BLOCK_HASH";
```

Comment:
This log line is a bit misleading when round is 1.   Will print in round 0 had next voted null block hash...